addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -233,16 +233,12 @@ const Wysiwyg = Widget.extend({
         this.call('bus_service', 'onNotification', this, (notifications) => {
             for (const [channel, busData] of notifications) {
                 if (
-                    channel[1] == 'editor_collaboration' &&
-                    channel[2] === modelName &&
-                    channel[3] === fieldName &&
-                    channel[4] === resId
+                    this._channelNames && this._channelNames.has(channel)
```

Comment:
Given `_channelNames` is synchronously defined below, under what condition would it be unset here? And if that's a concern why not just... define it in the `init`? In fact is there any reason not to define it in the `init`?